ports/rp2/boards/ADAFRUIT_KB2040/board.json

```diff
@@ -0,0 +1,23 @@
+{
+    "deploy": [
+        "../deploy.md"
+    ],
+    "docs": "",
+    "features": [
+        "Breadboard Friendly",
+        "Castellated Pads",
+        "RGB LED",
+        "SPI Flash",
+        "STEMMA QT/QWIIC",
+        "USB-C",
+        "Pink"
+    ],
+    "images": [
+        "5302-12.jpg"
```

Comment:
@qpucka  Could you please send a PR to https://github.com/micropython/micropython-media to add this file.

Reply:
@jimmo Done. https://github.com/micropython/micropython-media/pull/27